checkov/github/dal.py

```diff
@@ -10,14 +10,13 @@
 
 class Github(BaseVCSDAL):
     def __init__(self) -> None:
-        super().__init__()
-
         self.github_conf_dir_path = ""
         self.github_org_security_file_path = ""
         self.github_branch_protection_rules_file_path = ""
         self.github_org_webhooks_file_path = ""
         self.github_repository_webhooks_file_path = ""
         self.github_repository_collaborators_file_path = ""
+        super().__init__()
```

Comment:
can you add a comment the init has to be at the end and also add or extend an existing UT to check some/all attributes are filled with something?

Reply:
Hi! thanks for the review :) I've added a test for this (all paths should be initialized even if not used)
Also, I moved the type declaration (by Nimrod's suggestion) to be in the class instead of the init method.
💪 🐶